src/FPGA_common/FPGA_common.cpp

```diff
@@ -1224,25 +1166,31 @@ double FPGA::DetectRefClk(double fx3Clk)
     return clkTbl[i - 1];
 }
 
+/// @brief Gets the information about the gateware of the device from the FPGA.
+/// @return The gateware information of the FPGA.
 FPGA::GatewareInfo FPGA::GetGatewareInfo()
 {
     GatewareInfo info;
     info.boardID = 0;
     info.version = 0;
     info.revision = 0;
+    info.hardwareVersion = 0;
 
     const uint32_t addrs[4] = { 0x0000, 0x0001, 0x0002, 0x0003 };
     uint32_t data[4];
     if (ReadRegisters(addrs, data, 4) != OpStatus::SUCCESS)
         return info;
 
     info.boardID = data[0];
-    info.version = static_cast<int>(data[1]);
```

Comment:
The `version` field itself is an `int` so the conversion happens anyway; or would you prefer keeping this here in case the `lime::FPGA::GatewareInfo` structure changes?